source/macos/AdaptiveCards/AdaptiveCards/Views/ACRMultilineView.swift

```diff
@@ -0,0 +1,70 @@
+import AdaptiveCards_bridge
+import AppKit
+
+class ACRMultilineView: NSView, NSTextViewDelegate {
+    @IBOutlet var contentView: NSView!
+    @IBOutlet var scrollView: NSScrollView!
+    @IBOutlet var textView: ACRTextView!
+    
+    init() {
+        super.init(frame: .zero)
+        guard let bundle = Bundle(identifier: "com.test.test.AdaptiveCards") else {
+            logError("Bundle is nil")
+            return
+        }
+        bundle.loadNibNamed("ACRMultilineView", owner: self, topLevelObjects: nil)
+        setupViews()
+        setupConstaints()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setupViews() {
+        addSubview(contentView)
+    }
+    
+    private func setupConstaints() {
+        contentView.translatesAutoresizingMaskIntoConstraints = false
+        contentView.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
+        contentView.trailingAnchor.constraint(equalTo: trailingAnchor).isActive = true
+        contentView.topAnchor.constraint(equalTo: topAnchor).isActive = true
+        contentView.bottomAnchor.constraint(equalTo: bottomAnchor).isActive = true
+        heightAnchor.constraint(equalToConstant: 45.0).isActive = true
+    }
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        scrollView.borderType = NSBorderType.bezelBorder
+        scrollView.autohidesScrollers = true
+        textView.delegate = self
+    }
+    
+    fileprivate var placeholderAttrString: NSAttributedString?
+    var maxLen: Int = 0
+    
```

Comment:
Move these properties also to the begining of the class.
`private` can be used instead of `fileprivate`specifier